source/synthDrivers/_espeak.py

```diff
@@ -291,14 +319,17 @@ def initialize():
 		os.path.abspath("synthDrivers"),0)
 	if sampleRate<0:
 		raise OSError("espeak_Initialize %d"%sampleRate)
-	player = nvwave.WavePlayer(channels=1, samplesPerSec=sampleRate, bitsPerSample=16, outputDevice=config.conf["speech"]["outputDevice"])
+	player = nvwave.WavePlayer(channels=1, samplesPerSec=sampleRate, bitsPerSample=16,
+		outputDevice=config.conf["speech"]["outputDevice"],
+		buffered=True)
 	espeakDLL.espeak_SetSynthCallback(callback)
 	bgQueue = queue.Queue()
 	bgThread=BgThread()
 	bgThread.start()
+	onIndexReached = indexCallback

```

Comment:
Shouldn't 'onIndexReached` be set before `espeakDLL.espeak_SetSynthCallback(callback)`? `callback` can call `onIndexReached`.